src/styles/abstracts/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$width: 707px;
```

Comment:
The width can't be a fixed value. It has to be calculated basing on size of cell, number of cells, gaps